cfgov/unprocessed/apps/mmt-my-money-calendar/js/components/forms.js

```diff
@@ -0,0 +1,182 @@
+import clsx from 'clsx';
+import { useReducer, useCallback } from 'react';
+import { formatCurrency, toCents } from '../lib/currency-helpers';
+import { recurrenceRules, DAY_OPTIONS } from '../lib/calendar-helpers';
+
+import closeRound from '@cfpb/cfpb-icons/src/icons/close-round.svg';
+
+export const Checkbox = ({ id, name, onChange, checked, label, value = '1', ...props }) => {
+  const changeHandler = useCallback(
+    (evt) => {
+      evt.target.value = evt.target.checked;
+      onChange(evt);
+    },
+    [onChange]
+  );
+
+  return (
+    <div className="m-form-field m-form-field__checkbox">
+      <input
+        className="a-checkbox"
+        type="checkbox"
+        name={name}
+        id={id}
+        onChange={changeHandler}
+        checked={checked}
+        value={value}
+        {...props}
+      />
```

Comment:
This field has a label, but it might be flagged because the input and label use a prop to populate their `id` and `htmlFor` props, respectively. Also, React uses `htmlFor=""` rather than `for=""` on labels to avoid JS syntax collision issues.

The other fields that were flagged also all have labels.

Reply:
This is a false positive. The flagged React component does have a label, but I'm guessing that it's not being picked up because React uses `<label htmlFor="">` instead of `<label for="">` to avoid colliding with regular JS syntax, and I'm using a variable for the ID attribute value in the input's `id` and the label's `htmlFor`. 

The other comments from accesslint in this file are also false positives.